.github/workflows/dotnet-ci.yml

```
@@ -1,13 +1,15 @@
 jobs:
   restore:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v6
       - name: Setup .NET Core SDK
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v5
         with:
           dotnet-version: '10.x'
           include-prerelease: true
```

actionlint (1.7.10)

[error] input "include-prerelease" is not defined in action "actions/setup-dotnet@v5". available inputs are "cache", "cache-dependency-path", "config-file", "dotnet-quality", "dotnet-version", "global-json-file", "owner", "source-url"

(action)